docs/source/topics/proc_setting-up-codeready-containers.adoc

```diff
@@ -25,3 +25,31 @@ Always run the [command]`{bin}` executable with your user account.
 ----
 $ {bin} setup
 ----
+
+.Consent for telemetry data collection
+
+The `{bin} setup` command prompts you for optional, anonymous usage data collection to assist with development.
+No personally identifiable information is collected.
+
+* To skip the prompt, use the `--force` flag:
```

Comment:
We don't have `force` flag. So it is going to be prompted at least once, if user chose not to activate telemetry then `consent-telemetry` will be set to `no` in the config.

```
$ ./crc setup
CodeReady Containers is constantly improving and we would like to know more about usage!
Would you like to contribute anonymous usage statistics (more details at https://developers.redhat.com/article/tool-data-collection)? [y/N]: n
No worry, you can still enable telemetry manually with the command 'crc config set consent-telemetry yes'.
...

$ ./crc config view
- consent-telemetry                     : no
```

And
```
$ ./crc setup
CodeReady Containers is constantly improving and we would like to know more about usage!
Would you like to contribute anonymous usage statistics (more details at https://developers.redhat.com/article/tool-data-collection)? [y/N]: y
Thanks for helping us! You can disable telemetry with the command 'crc config set consent-telemetry no'.
...

$ ./crc config view
- consent-telemetry                     : yes
```

Reply:
Removed the "skip prompt" section. Thanks!